### DISCUSSION TITLE
关于日志记录的疑问

### DISCUSSION THREAD

**Discussion by zqb666** [zuihou/zuihou-admin-cloud#253] [02.提问]
我新建的服务（按照lamp-generator生成的）  我在业务服务接口里添加日志注解
<img width="770" alt="image" src="https://user-images.githubusercontent.com/6092525/227169707-be9054ef-e1b9-4c89-8d63-edfbf65eda75.png">
发现这个接口调用的时候不会插入日志 怎么让它插入日志？

貌似 SysLogListener saveSysLog方法没进去

**Comment by zuihou** [comment]
调试：SysLogAspect 、XxxWebConfiguration 和 SysLogListener

<img width="1077" alt="image" src="https://user-images.githubusercontent.com/8500410/227178227-95161706-4370-4c5d-9479-03c8f5580c7a.png">

**Reply by zqb666** [reply]
<img width="870" alt="image" src="https://user-images.githubusercontent.com/6092525/227179857-794fccfc-d138-4998-a169-14930bb2806d.png">
这里mei'jin'l没进来

**Reply by zuihou** [reply]
启动程序时才会进来，前提是 注解上的条件要生效

**Reply by zqb666** [reply]
问题是 authority那边 操作记录日志 我新建的就不行 是不是少了 什么配置或者依赖？

**Reply by zqb666** [reply]
问题是 authority那边 操作记录日志可以记录用户的一些操作日志  我新建业务记录不了日志   是不是少了 什么配置或者依赖？

**Reply by zqb666** [reply]
<img width="1278" alt="image" src="https://user-images.githubusercontent.com/6092525/227180831-1842cc5b-6595-4d56-968f-77953429488a.png">
已经到这一步了 为什么 监听那边不保存？

**Reply by zqb666** [reply]
<img width="1024" alt="image" src="https://user-images.githubusercontent.com/6092525/227181775-13865987-a07b-4d30-97ec-e2df7eeb7bc8.png">
启动的时候进来了 能看出什么？

**Comment by zuihou** [comment]
https://user-images.githubusercontent.com/6092525/227181775-13865987-a07b-4d30-97ec-e2df7eeb7bc8.png


这个代码要在你新建的服务里面启动，并确保执行！！！！！！！！！！！

不是authority服务！

**Reply by zuihou** [reply]
每个服务都要在启动时通过@Bean 实例SysLogListener
